extensions/objects/templates/position-of-terms-editor.component.ts

```diff
@@ -52,6 +52,10 @@ export class PositionOfTermsEditorComponent implements OnInit {
         this.localValue = this.positionOfTerms[i] as unknown as PositionOfTerm;
       }
     }
+    if (!this.value) {
```

Comment:
Why this `if` differs from the other places?

Reply:
Fixed.